content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -90,7 +90,7 @@ The following section demonstrates how to query an explicit many-to-many relatio
 The following query:
 
 1. Creates a `Post`
-2. Creates a category assigment, or `CategoriesOnPosts` (assigned by Bob, assigned today)
+2. Creates a category assignment, or `CategoryToPost` (assigned by Bob, assigned today)
```

Comment:
I find this phrasing not very clear, especially for a beginner. The "or `CategoryToPost` could be read as an _actual_ or (as in _either ... or ..._) but not as (what I assume is intended) representing a different phrasing or what a "category assignment" is on a technical level. Can you try to clarify this? 

(I hope my explanation makes sense, if no, feel free to ask for more clarification. Also, it seems that you just [reused existing phrasing](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#querying-an-explicit-many-to-many) so I'd ask you to update the original phrasing of this as well 🙏 )

Reply:
I clarified what the relation table is and explicitly called out that `CategoriesOnPosts` as a relation table in each of the examples.